File: common/src/main/java/me/reidj/forest/protocol/SaveUserPackage.java
package me.reidj.forest.protocol;

import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.RequiredArgsConstructor;
import ru.cristalix.core.network.CorePackage;
import stat.Stat;

import java.util.UUID;

@Data
@EqualsAndHashCode(callSuper = false)
@RequiredArgsConstructor
public class SaveUserPackage extends CorePackage {

    // request
    private final UUID user;
    private final Stat stat;

    // no response

}

File: common/src/main/kotlin/me/reidj/forest/protocol/SaveUserPackage.kt
package me.reidj.forest.protocol

import ru.cristalix.core.network.CorePackage
import me.reidj.forest.data.Stat
import java.util.*

/**
 * @project : forest
 * @author : Рейдж
 **/
data class SaveUserPackage(var uuid: UUID, var stat: Stat): CorePackage()
